src/Vm.sol

```diff
@@ -134,12 +140,12 @@ interface VmSafe {
     // decode the tuple in that order, and thus fail.
 
     // Returns the RPC url for the given alias
-    function rpcUrl(string calldata) external returns (string memory);
+    function rpcUrl(string calldata) external view returns (string memory);
     // Returns all rpc urls and their aliases `[alias, url][]`
-    function rpcUrls() external returns (string[2][] memory);
+    function rpcUrls() external view returns (string[2][] memory);
 
     // If the condition is false, discard this run's fuzz inputs and generate new ones.
-    function assume(bool) external;
+    function assume(bool) pure external;
```

Comment:
_(arbitrary comment location to start a thread)_

A few cheatcodes in `Vm` seem like they can be view too? I think just `isPersistent` and `activeFork`

Reply:
Yes, will update!